conf/rh_cloud.yaml.template

```diff
@@ -12,6 +12,3 @@ RH_CLOUD:
     STAGE_USERNAME:
     TOKEN:
     STAGE_TOKEN:
-    SETUP_SCRIPT:
-    SATELLITE_IOP_REPO:
-    IMAGE_PATHS:
```

Comment:
is it possible to keep the IMAGE_PATHS setting here, as it could be useful to test any upstream IoP changes with konflux images in future?